docs/source/cuml-accel/limitations.rst

```diff
@@ -443,6 +443,40 @@ Additionally, the following fitted attributes are currently not computed:
 
 - ``n_iter_``
 
+LinearSVC
+^^^^^^^^^
+
+``LinearSVC`` will fall back to CPU in the following cases:
+
+- If ``X`` is sparse.
+
+The following fitted attributes are currently not computed:
+
+- ``n_iter_``
+
+Additional notes:
+
+- Sample weight functionality may not produce equivalent results to replicating data according to weights.
+- Models may not be picklable; pickling or unpickling may fail.
+- Error messages and formats may differ from scikit-learn, including C++ stack traces.
```

Comment:
This (error messages not matching, not C++ stack traces) is pretty true across all of `cuml.accel`. Not sure if it's worth mentioning specifically on these estimators, I'd probably move this to the top-level caveat list (and add a "sometimes" before "including" to soften it a bit).